### ISSUE TITLE
Bug: latest commit 839be0a is an undocumented breaking change

### ISSUE THREAD

**Issue by chrisgrieser** [johmsalas/text-case.nvim#49]
cc @peterfication 

Commit 839be0a introduces a breaking change for almost all methods of this plugin, since the "description" apparently also determines the function names when calling the methods via lua API.

This breaking change occurred without any corresponding change in the readme, and without using a `break!` in the commit message. 

Tbf, since there is no real benefit to the commit other than aesthetics, I'd suggest to simply revert it, as it is not worth it to break people's configs over that?

**Comment by johmsalas** [comment]
Thanks for the report. Will have a look when I get back next week. The commit was reversed

**Comment by peterfication** [comment]
Oh no this is bad. @chrisgrieser I'm sorry for your inconvenience. You're right, it's not worth to make a breaking change because of that, especially because the way of calling the methods would be really cumbersome. My bad that I missed this.

I created another PR that handles it more gracefully. The discussion of whether this PR is worth it or not can be continued there. So this issue can be closed as the initial commit has been reverted.
